src/acceptance-test/java/org/zalando/nakadi/webservice/CursorsServiceAT.java

```diff
@@ -180,6 +174,44 @@ public void whenFirstCursorIsNotCommittedThenNextCursorsAreNotSkipped() throws E
     }
 
     @Test
+    public void whenMultipleCursorsForSamePartitionThenResultsAreCorrect() throws Exception {
+        CURATOR.setData().forPath(offsetPath(P1), "000100".getBytes(UTF_8));
```

Comment:
why do not you want to use 18 digits offset, the offset we decided to use in Nakadi?